datastore-translator/translator.go

```diff
@@ -117,7 +117,18 @@ func DatastoreEntityToProtoMessage(src *datastore.Entity, dst proto.Message, sna
 	return err
 }
 
-func toDatastoreValue(fValue reflect.Value, snakeCase bool) (*datastore.Value, error) {
+func toDatastoreValue(fName string, fValue reflect.Value, snakeCase bool, excludeFromIndex []string) (*datastore.Value, error) {
+	// NOTE: excludeFieldFromIndex needs to contain original Protobuf model field names which can also include underscores
```

Comment:
For consistency, I think it's the best if user passes in original field names as defined on the Protobuf message which can also contain underscores.